littlepay/main.py

```diff
@@ -64,6 +64,8 @@ def _maincmd(name, help):
     groups_link = _subcmd(groups_commands, "link", help="Link one or more concession groups to a product")
     groups_link.add_argument("product_id", help="The ID of the product to link to")
 
+    _subcmd(groups_commands, "migrate", help="Migrate a group from the old Customer Group format to the current format")
```

Comment:
Can we also take into account the `--force` flag for this sub command?

Thinking that since it operates on a potentially large number of groups based on filtering/not filtering, and makes changes to the Littlepay config (vs. just listing existing data).

Reply:
Great idea, I will add

Reply:
Added test and support for `--force` flag in [1c94d66](https://github.com/cal-itp/littlepay/pull/49/commits/1c94d66e153bf09068e0be8aa14d6a3e5a8abc08)

Reply:
Ahh sorry. I think I missed some pieces in the implementation. Putting PR into draft

Reply:
Added tests and support for `--force` flag in [056b8f3](https://github.com/cal-itp/littlepay/pull/49/commits/056b8f38437073e7cf88d6e62bdbd76c810d2b1f) and [4c691b1](https://github.com/cal-itp/littlepay/pull/49/commits/4c691b1d95f6e0c5ebc53b53453950f4043bb057)